README.md

```diff
@@ -1,71 +1,90 @@
+## Download and Installation
+To use the bridge, you'll need:
+- The latest pre-built Windows Bridge for iOS SDK, that you can download [here](https://aka.ms/winobjc_latest) (click on the **winobjc.zip** file under *Downloads* to start the download and extract it to the local directory of your choice when complete).
+- **Windows 10**, build 10586 or higher (validate your version number [here](http://windows.microsoft.com/en-US/windows/which-operating-system)).
+- **Visual Studio 2015 Update 3** with Windows developer tools. Visual Studio 2015 Community is available for free [here](https://dev.windows.com/downloads). Select (at least) the following components during installation:
+   1. Programming Languages -> Visual C++
+   2. Universal Windows App Development Tools (all)
+   
+If you want to build the bridge from source instead, you'll also need to install [Git LFS](https://git-lfs.github.com) before cloning the repo. For more detailed instructions, see the [Building From Source](https://github.com/Microsoft/WinObjC/wiki/Building-From-Source) wiki page.
 
-**Do not use the Download ZIP option via the "Clone or download" button above.** You will not be able to build the bridge using the resulting .zip file. See ["I'm getting errors after downloading the bridge through the "Download ZIP" button on Github. What should I do?"](https://github.com/Microsoft/WinObjC/wiki/FAQ#im-getting-errors-after-downloading-the-bridge-through-the-download-zip-button-on-github-what-should-i-do) for more information.
+If you're having trouble downloading & installing the bridge, see the [Known Issues](https://github.com/Microsoft/WinObjC#known-issues) section below and check our [FAQ](https://github.com/Microsoft/WinObjC/wiki/FAQ).
 
-# Welcome to the Windows Bridge for iOS project preview
-The Windows Bridge for iOS (also referred to as WinObjC) is a Microsoft open-source project that provides an Objective-C development environment for Visual Studio and support for iOS APIs. The bridge allows you to create Universal Windows Platform (UWP) apps that will run on many Windows devices using iOS APIs and Objective-C code alongside Windows 10 features like Cortana and Live Tiles.
+## Getting Started With the Bridge
+### Importing your Xcode Project
+When using the bridge, the first thing you'll want to do is generate a Visual Studio solution from your Xcode project:
 
+1. Open the Windows Command Prompt (type **cmd** in the Start Menu to find it) and navigate to your Xcode project directory using the **cd** command. Note that you should go to the directory containing your project *.xcworkspace* or *.xcodeproj* folder: you shouldn't be inside those folders, but one level above them.
```

Comment:
I might suggest leaving the first sentence, but replace the second with:

**Note:** Use the directory containing your .workspace or .xcodeproj folder.

Sometimes too much direction is more confusing than too little.